addons/account/wizard/account_automatic_entry_wizard.py

```diff
@@ -203,6 +204,7 @@ def _get_move_dict_vals_change_account(self):
                     'partner_id': partner.id or None,
                     'currency_id': currency.id,
                     'amount_currency': (account_balance > 0 and -1 or 1) * abs(account_amount_currency),
+                    'analytic_distribution': analytic_distribution and dict(analytic_distribution),
```

Comment:
Since it's now a `frozendict`, 
```suggestion
                    'analytic_distribution': analytic_distribution,
```